i18next-parser.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   createOldCatalogs: false,
   input: ['./src/**/*.{tsx,ts}'],
   keepRemoved: true,
-  keySeparator: false,
+  keySeparator: '/',
```

Comment:
Hopefully this will not conflict with any future translations, but just in case we can also use a more exotic symbol. Possible a backslash \?

Reply:
If we go this way, I think we should find another, less commonly used symbol.
Another option is to keep the translations list flat (as today) and use `aria/*` or `aria.*` as a prefix. What do you think about this?

Reply:
I agree that separator should be a bit more crazy. Otherwise I approve the PR.

Reply:
I like the idea of keeping the translations list flat. So I'm going to keep the `aria/` prefix, but disable the key separator in i18next 👍